test/functional/ui/ext_win_spec.lua

```diff
@@ -0,0 +1,301 @@
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local spawn, set_session, clear = helpers.spawn, helpers.set_session, helpers.clear
+local feed, command = helpers.feed, helpers.command
+local eq, iswin = helpers.eq, helpers.iswin
+
+describe('In ext-win mode', function()
+  local screen
+  local nvim_argv = {helpers.nvim_prog, '-u', 'NONE', '-i', 'NONE', '-N',
+                     '--cmd', 'set shortmess+=I background=light noswapfile belloff= noshowcmd noruler',
+                     '--embed'}
+
+  before_each(function()
+    local screen_nvim = spawn(nvim_argv)
+    set_session(screen_nvim)
+    screen = Screen.new(15, 5)
+    -- TODO(utkarshme): should not have to set ext_multigrid
+    screen:attach({ext_windows=true, ext_multigrid=true})
```

Comment:
@bfredl The tests start failing if I remove `ext_multigrid=true` from this line. Whereas it works in the python-gui without setting it explicitly.

Reply:
hmm strange, do you set that `ext_windows` imply `ext_multigrid`, if so, where?

Reply:
This line: https://github.com/UtkarshMe/neovim/pull/2/files#diff-acdca5aaf6a1d3676bd25ca27959b985R117

Reply:
hmm looks correct to me, can look into it more tomorrow if still a problem.

Reply:
Roger that!